profile/README.md

```diff
@@ -1 +1,18 @@
 # The Space ROS Organization
+
+<img src="https://github.com/space-ros/space-ros/blob/985f465ad0ff484c9d589e22d4628bde81b38661/logos/spaceros_white_on_blue.png" alt="Space ROS Logo - White on Blue" width="400"/>
+
+This is the home of Space ROS development.
+
+<p align="justify">
+Space ROS is an open-source software framework, derived from ROS 2 that will be hardened to be compatible with the demands of safety-critical space robotics applications. Conforming to the ROS 2 Application Programming Interface (API), Space ROS is designed to be platform independent, portable, and project independent. Space ROS will provide a robust framework for space robotics applications where ROS 2 applications can be reused with little to no modification, enabling the space community to take advantage of the innovation of the ROS community. This will shorten the time for development of novel space robotics capabilities, enable reuse of capabilities between missions, and lower the life-cycle cost of new space robotics missions.
```

Comment:
Let's do one sentence per line for the sake of diffing:
```suggestion
Space ROS is an open-source software framework, derived from ROS 2 that will be hardened to be compatible with the demands of safety-critical space robotics applications.
Conforming to the ROS 2 Application Programming Interface (API), Space ROS is designed to be platform independent, portable, and project independent.
Space ROS will provide a robust framework for space robotics applications where ROS 2 applications can be reused with little to no modification, enabling the space community to take advantage of the innovation of the ROS community.
This will shorten the time for development of novel space robotics capabilities, enable reuse of capabilities between missions, and lower the life-cycle cost of new space robotics missions.
```